generator/package.json

```diff
@@ -2,8 +2,8 @@
 	"dependencies": {
 		"graceful-fs": "latest",
```

Comment:
if libraries need to be changed to fixed ones - why not change them all?

Reply:
I used the same libraries we use in PQD. If you happen to know the right version for this one I will do the change in both branches :)